### ISSUE TITLE
Is this not implemented exception expected for LSTM?

### ISSUE THREAD

**Issue by serp777** [SciSharp/TensorFlow.NET#707]
Hello,

   Is LSTM in sequential supported currently?

System.NotImplementedException: ''

   at Tensorflow.Keras.Engine.Layer.Call(Tensors inputs, Tensor state, Boolean is_training)
   at Tensorflow.Keras.Layers.LSTM.Call(Tensors inputs, Tensor state, Boolean is_training)
   at Tensorflow.Keras.Engine.Layer.<>c__DisplayClass65_0.<FunctionalConstructionCall>b__0(NameScope scope)
   at Tensorflow.Binding.tf_with[T](T py, Action`1 action)
   at Tensorflow.Keras.Engine.Layer.FunctionalConstructionCall(Tensors inputs)
   at Tensorflow.Keras.Engine.Layer.Apply(Tensors inputs, Tensor state, Boolean is_training)
   at Tensorflow.Keras.Engine.Sequential.add(Layer layer)

Associated code that is triggering this:
`var layers = new LayersApi();
            this.rnnModel = new Sequential(new SequentialArgs());
            this.rnnModel.add(layers.Input(new Shape(this.shapeBatchSize, this.shapeNumDataPoints)));
            Activations activations = new Activations();
            
            this.rnnModel.add(layers.LSTM(lstmLayers[0].NumNodes, recurrent_activation: activations.Sigmoid, activation: lstmLayers[0].ActivationFunction, return_sequences: true));`

Thanks

**Comment by serp777** [comment]
Looks like the API just hasn't been fully implemented here for LSTM sequential

**Comment by serp777** [comment]
bump?

**Comment by Oceania2018** [comment]
Not yet, havn't chance to implement it.

**Comment by serp777** [comment]
> Not yet, havn't chance to implement it.

Any idea how difficult this would be for someone to implement who isn't familiar with the tensorflow.net code base but has a lot of experience with C#? Like is it realistic for a tensorflow.net novice to try this feature and create a pull request?

**Comment by Oceania2018** [comment]
@serp777 You should try your best, I don't know how difficult it is for you. We all welcome the PRs and glad to help.

**Comment by serp777** [comment]
ok great ty I will give it a shot

**Comment by GeorgeS2019** [comment]
@serp777 please keep posting your progress here. There have been in the past talented Microsoft developers contributed to this repository. By doing what you plan to do, you may learn a lot from many talented .NET developers.
